rails_application/Gemfile.lock

```diff
@@ -14,75 +15,77 @@ PATH
     math (1.0.0)
       rake
 
-GEM
-  remote: https://oss:7AXfeZdAfCqL1PvHm2nvDJO6Zd9UW8IK@gem.mutant.dev/
-  specs:
-    mutant-license (0.1.1.2.1627430819213747598431630701693729869473.4)
-
```

Comment:
it seems you've accidentally changed remotes in `Gemfile.lock` files. This is probably caused by an older Bundler version. Can you please upgrade your `bundler` and ensure those remotes remain unchanged?